components/admin_console/group_settings/group_details/group_details.jsx

```diff
@@ -235,6 +298,13 @@ export default class GroupDetails extends React.PureComponent {
                     </div>
                 </div>
 
+                <SaveChangesPanel
+                    saving={saving}
+                    cancelLink='/admin_console/user_management/teams'
```

Comment:
Should cancelling redirect us to teams? Or groups?

Reply:
probably groups, thanks for pointing out.